pkg/appdefinition/acornfile-schema.acorn

```diff
@@ -215,12 +215,15 @@
 		match "probes|probe":                     Probes
 		match "depends[oO]n|depends_on|consumes": string || [string]
 		match "mem|memory":                       int
+		user?:																		User
```

Comment:
nit: the spacing here looks a little off.

Reply:
Tabs vs Spaces >.<